src/word_splitters.rs

```diff
@@ -204,9 +205,10 @@ where
             if prev < word.word.len() || prev == 0 {
                 let w = Word {
                     word: &word.word[prev..],
-                    width: display_width(&word[prev..]),
-                    whitespace: word.whitespace,
-                    penalty: word.penalty,
+                    // see comment by display_width call above regarding
```

Comment:
I think "above" is a bit far away here. Also, it looks like `tab_width` is readily available here so we could just use it?

Reply:
You're right, I think I just forgot to update this call when I updated the one above. Fixed in a901fce7b0165ba2186aad3e5fdca02b96976949.